assets/js/sections/contesting.js

```diff
@@ -208,20 +214,35 @@ $('#start_date').change(function () {
 
 // On Key up check and suggest callsigns
 $("#callsign").keyup(function () {
-	var call = $(this).val();
+	var call = $(this).val().toUpperCase();
```

Comment:
if you use `let` instead of `var ` here you can re-use the `call` variable in the scope below instead of re-defining `call`

Reply:
I chose to not reuse `call` on purpose. The moment the success-callback is executed, the value may have changed, and the 'cache' may be invalidated already, or only a subset of the 'cached callsigns' should be displayed. 

But yes, lets go with `let` here.